From: Gregorio Guidi <gregorio.guidi@gmail.com>

Retain the old behavior of the build system even with automake 1.11.2.

Reported-by: Oleg Bulatov <dmage@bk.ru>
X-Gentoo-Bug: 397429
X-Gentoo-Bug-URL: https://bugs.gentoo.org/show_bug.cgi?id=397429
X-KDE-Bug: 290719
X-KDE-Bug-URL: https://bugs.kde.org/show_bug.cgi?id=290719
Signed-off-by: Anthony G. Basile <blueness@gentoo.org>

diff -Nur valgrind-3.7.0.orig/coregrind/Makefile.am valgrind-3.7.0/coregrind/Makefile.am
--- valgrind-3.7.0.orig/coregrind/Makefile.am	2011-10-27 12:55:06.000000000 +0200
+++ valgrind-3.7.0/coregrind/Makefile.am	2012-01-06 17:23:48.717115986 +0100
@@ -495,7 +495,8 @@
 	m_gdbserver/powerpc-altivec64l.xml
 
 # so as to make sure these get copied into the install tree
-pkglib_DATA  = $(GDBSERVER_XML_FILES)
+vgdatadir = $(pkglibdir)
+vgdata_DATA  = $(GDBSERVER_XML_FILES)
 
 # so as to make sure these get copied into the tarball
 EXTRA_DIST  += $(GDBSERVER_XML_FILES)
diff -Nur valgrind-3.7.0.orig/Makefile.am valgrind-3.7.0/Makefile.am
--- valgrind-3.7.0.orig/Makefile.am	2011-10-26 23:24:45.000000000 +0200
+++ valgrind-3.7.0/Makefile.am	2012-01-06 17:23:17.445114847 +0100
@@ -61,7 +61,8 @@
 # default.supp, as it is built from the base .supp files at compile-time.
 dist_noinst_DATA = $(SUPP_FILES)
 
-pkglib_DATA = default.supp
+vgdatadir = $(pkglibdir)
+vgdata_DATA = default.supp
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = valgrind.pc
